react/components/SearchResult.js

```diff
@@ -155,13 +156,11 @@ class SearchResult extends Component {
     return (
       <LoadingOverlay loading={showLoading && showLoadingAsOverlay}>
         <div className="vtex-search-result vtex-p pv5 ph3">
+          {mobile && (<div className="vtex-search-result__divider bt bw1 b--muted-4 w-100"></div>)}
```

Comment:
Remove the `bw1` to improve the design cohesion

Reply:
This line has been removed.